src/lib/containers/TotalQueryResults.tsx

```diff
@@ -188,47 +199,89 @@ const TotalQueryResults: FunctionComponent<TotalQueryResultsProps> = ({
     calculateTotal()
   }, [parentLoading, accessToken, lastQueryRequest])
 
-  const removeFacetSelection = ({
-    facet,
-    selectedValue,
-  }: FacetWithSelection) => {
+  const removeFacetSelection = (facetWithSelection: FacetWithSelection) => {
+    const { facet, selectedValue } = facetWithSelection
     if (facet.facetType === 'enumeration') {
       applyChangesToValuesColumn(
         lastQueryRequest,
         facet as FacetColumnResultValues,
-        applyChanges!,
+        applyChanges,
         selectedValue?.value,
         false,
       )
     } else {
       applyChangesToRangeColumn(
         lastQueryRequest,
         facet as FacetColumnResultRange,
-        applyChanges!,
+        applyChanges,
         [RadioValuesEnum.ANY, RadioValuesEnum.ANY],
       )
     }
   }
 
-  const removeSearchQuerySelection = (columnName: string, value: string) => {
+  const removeColumnSearchQuerySelection = (
+    columnName: string,
+    value: string,
+  ) => {
```

Comment:
Now. QueryFilters may not apply to specific columns, so this function must become more specific.

Reply:
The `(column, value)` signature remains because we still split an array of QueryFilter values into individual pills in the UI.